api/chalicelib/services/user_service.py

```diff
@@ -88,6 +88,7 @@ def find_users_to_notify_for_location(location: dict) -> Optional[List[dict]]:
 
     users = []
     zipcode_distances = find_zipcodes_in_radius(coordinates[0], coordinates[1], 50)
+    updated_since_limit = ms_since_epoch() - 1814400000 # 21 days in milliseconds
```

Comment:
This can actually be updated to be `ms_since_epoch(days=-21)` but I'll do that when I merge it in.